apps/judicial-system/backend/src/app/modules/case/interceptors/case.interceptor.ts

```diff
@@ -7,12 +7,23 @@ import {
   NestInterceptor,
 } from '@nestjs/common'
 
+import { Defendant, DefendantEventLog } from '../../defendant'
 import { Case } from '../models/case.model'
 import { CaseString } from '../models/caseString.model'
 
+export const transformDefendants = (defendants?: Defendant[]) => {
+  return defendants?.map((defendant) => ({
+    ...defendant.toJSON(),
+    sentToPrisonAdminDate: DefendantEventLog.sentToPrisonAdminDate(
```

Comment:
Just a thought, should we return this date as empty if isSentToPrisonAdmin === false? In cases where it was sent and withdrawn? Or are we fine with just keeping in mind everywhere that even if there's a date in this field it still might not currently really be sent to the prison?

Another thought, if we don't do that, should we then consider also storing an event when an indictment is withdrawn ? So we could display both dates if needed. (ex. Sent on 01.01.2024, withdrawn on 02.01.2024) Because otherwise it seems pointless for us to expose this date if isSentToPrisonAdmin === false

Reply:
I wouldn't want to change it in this PR though cause it's not a big deal having it like this, just feels like it might be less error prone to only return it when it's actually relevant

Reply:
Very good idea. I can easily change it now so I will.